libraries/wasm/package-lock.json

```diff
@@ -0,0 +1,5870 @@
+{
```

Comment:
This file should also be removed since it looks like you're using Yarn instead of NPM